dataland-community-manager/src/test/kotlin/org/dataland/datalandcommunitymanager/email/BulkDataRequestMessageBuilderTest.kt

```diff
@@ -0,0 +1,112 @@
+package org.dataland.datalandcommunitymanager.email
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+import org.dataland.datalandcommunitymanager.model.dataRequest.BulkDataRequest
+import org.dataland.datalandmessagequeueutils.cloudevents.CloudEventMessageHandler
+import org.dataland.datalandmessagequeueutils.constants.ExchangeName
+import org.dataland.datalandmessagequeueutils.constants.MessageType
+import org.dataland.datalandmessagequeueutils.constants.RoutingKeyNames
+import org.dataland.datalandmessagequeueutils.messages.InternalEmailMessage
+import org.dataland.keycloakAdapter.auth.DatalandJwtAuthentication
+import org.dataland.keycloakAdapter.auth.DatalandRealmRole
+import org.dataland.keycloakAdapter.utils.AuthenticationMock
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.springframework.security.core.context.SecurityContext
+import org.springframework.security.core.context.SecurityContextHolder
+import java.util.*
+class BulkDataRequestMessageBuilderTest {
+    val objectMapper = jacksonObjectMapper()
+    private lateinit var authenticationMock: DatalandJwtAuthentication
+    private val cloudEventMessageHandlerMock = Mockito.mock(CloudEventMessageHandler::class.java)
+    private val correlationId = UUID.randomUUID().toString()
+    private val bulkDataRequest = BulkDataRequest(
+        companyIdentifiers = setOf(
+            "AR8756188701," +
+                "9856177321",
+            "28f05156-e1ba-1ea8-8d1e-d4833f6c7afgh",
+        ),
+        dataTypes = setOf(DataTypeEnum.p2p, DataTypeEnum.lksg),
+        reportingPeriods = setOf("2020, 2023"),
+    )
+    private val acceptedCompanyIdentifiers = listOf("AR8756188701,9856177321")
+    fun formatReportingPeriods(reportingPeriods: Set<String>) =
+        reportingPeriods.toList().sorted().joinToString(", ")
+
+    @BeforeEach
```

Comment:
Wieso eigentlich nicht nur "Before".
Es reicht doch wahrscheinlich wenn das einmal initialisiert wird, oder?

Reply:
In der version wie sie hier steht. Absolut richtig. 
Ich hab jetzt noch weitere Mocks hinzugefügt. Die sind aber auch Konstant.
Macht aber bei nur einem Test auch keinen Unterschied, daher lasse ich es so.

Reply:
Jup done.